tests/vmi_multus_test.go

```diff
@@ -387,6 +373,38 @@ var _ = Describe("Multus Networking", func() {
 		})
 	})
 
+	Context("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance with ovs-cni plugin interface and custom MAC address.", func() {
+		interfaces := []v1.Interface{{Name: "default", InterfaceBindingMethod: v1.InterfaceBindingMethod{Bridge: &v1.InterfaceBridge{}}},
+			{Name: "ovs", InterfaceBindingMethod: v1.InterfaceBindingMethod{Bridge: &v1.InterfaceBridge{}}}}
+		networks := []v1.Network{{Name: "default", NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}}},
+			{Name: "ovs", NetworkSource: v1.NetworkSource{Multus: &v1.CniNetwork{NetworkName: "ovs-net-vlan100"}}}}
+		ovsIfIdx := 1
+
+		BeforeEach(func() {
+			// Create a VM with ovs-cni network interface and a default MAC address.
+			vmiTwo = createVMI(interfaces, networks)
+			tests.WaitUntilVMIReady(vmiTwo, tests.LoggedInAlpineExpecter)
+
+			configInterface(vmiTwo, "eth1", "10.1.1.2/24", "localhost:~#")
+		})
+
+		AfterEach(func() {
+			deleteVMIs(virtClient, []*v1.VirtualMachineInstance{vmiOne, vmiTwo})
+		})
+
+		It("[test_id:676]should configure valid custom MAC address on ovs-cni interface.", func() {
+			By("Creating a VM  with custom MAC address on its ovs-cni interface.")
+			interfaces[ovsIfIdx].MacAddress = "50:00:00:00:90:0d"
+			vmiOne = createVMI(interfaces, networks)
+			tests.WaitUntilVMIReady(vmiOne, tests.LoggedInAlpineExpecter)
+
+			By("Configuring static IP address to the ovs interface.")
+			configInterface(vmiOne, "eth1", "10.1.1.1/24", "localhost:~#")
+
+			By("Ping from the VM with the custom MAC to the other VM.")
+			pingVirtualMachine(vmiOne, "10.1.1.2", "localhost:~#")
```

Comment:
It will be also nice to check the interface mac running `ip link eth1` inside the vm to see the mac address

Reply:
Done